Sources/SwiftProtobuf/ProtoNameProviding.swift

```diff
@@ -42,8 +34,12 @@ extension ProtoNameProviding {
   public func _protobuf_fieldNames(for number: Int) -> FieldNameMap.Names? {
     return Self._protobuf_fieldNames.fieldNames(for: number)
   }
+}
 
-  public func _protobuf_fieldNumber(forProtoName name: String) -> Int? {
-    return Self._protobuf_fieldNames.fieldNumber(forProtoName: name)
-  }
+
+extension ProtoNameProviding where Self: ExtensibleMessage {
```

Comment:
Ooops.  That was an experiment to see if I could move some of the overrides out of the generated code.   I'll remove it.